### ISSUE TITLE
`bigtools bedgraphtobigwig` -- add option to skip invalid chromosomes

### ISSUE THREAD

**Issue by nleroy917** [jackh726/bigtools#56]
`wigToBigWig` has an option `-clip` to simply issue a warning instead of crashing when a chromosome not represented in `chrom.sizes` file is found in the `.wig` file:

```
-clip - If set just issue warning messages rather than dying if wig
                  file contains items off end of chromosome or chromosomes
                  that are not in the chrom.sizes file.
```

Is it possible to have similar functionality in `bedgraphtobigwig`? I would imagine a similar flag like `--clip`

**Comment by jackh726** [comment]
Seems simple enough :) I'll add this over the weekend.

**Comment by nleroy917** [comment]
Thanks a lot! I’d love to take a crack at it, since I’m currently learning Rust.. hadn't gotten a chance to dive too deeply, though.

**Comment by jackh726** [comment]
Please feel free :) Very happy to answer questions or guide you if you need!

**Comment by jackh726** [comment]
@nleroy917 are you still interested in this?

**Comment by nleroy917** [comment]
Yes! Sorry I wasnt able to get to it this weekend, but this week I should! Unless you are in a hurry to close the issue which is understandable!

**Comment by jackh726** [comment]
No, you are more than welcome to it :) no rush from me
